lib/common/utils.ts

```diff
@@ -158,6 +158,11 @@ export function patchProperty(obj: any, prop: string, prototype?: any) {
     return;
   }
 
+  const onPropPatchedSymbol = zoneSymbol('on' + prop + 'patched');
+  if (obj[onPropPatchedSymbol]) {
+    // return;
```

Comment:
please remove the comment.

Reply:
Done. The comment was to see if tests are OK in case of no restriction

Reply:
Yes, judging by heap usage log - memory leak disappeared both in test repo and in huge production project.
Despite the fact that Jest with --detectLeaks reports there are still leaks, again, judging by the heap usage log they are minor.
I have attached log with more accurate measurement for test repo as PR comment here, please take a look.

So I think this solution is viable, provided we fix tests' failures for PR  in this (zone.js) repo.